gammapy/maps/axes.py

```diff
@@ -2973,6 +2973,56 @@ def to_header(self, format="gadf", idx=0):
 
         return header
 
+    def group_table(self, edges):
+        """Compute bin groups table for the TimeMapAxis, given coarser bin edges.
+
+        Parameters
+        ----------
+        edges : list of `~astropy.time.Time` or `~astropy.units.Quantity`
+            Start and stop time for each interval to compute the LC.
+
+        Returns
+        -------
+        groups : `~astropy.table.Table`
+            Group table.
+        """
+
+        for _, edge in enumerate(edges):
+            if not isinstance(edge, Time):
+                edges[_] = self.reference_time + edges[_]
+
+        time_intervals = [(edges[i], edges[i + 1]) for i in range(len(edges) - 1)]
```

Comment:
Here you are assuming contiguous edges. That need not be, no?

Reply:
Whoops, you are totally right! `TimeMapAxis` is interval-based so this should be reflected. Fixed in the most recent commit